### PR TITLE
PAYEE-940 update list content on tile

### PR DESCRIPTION
**Pull request by masahmad**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by Jake-Raffe — APPROVED** [review]

File: app/views/includes/forms/card.scala.html [hunk 1]
```diff
@@ -37,7 +39,16 @@
         @heading
       }
     </@headingTag>
-    <p> @bodyContent</p>
+    @if(isBodyContentList){
```
Comment (brendanstraughan) [inline app/views/includes/forms/card.scala.html:42]: not sure about this so @sergi-hmrc correct me if wrong but would doing 
@if(bodyContentList.nonEmpty)
be better as then we wouldnt have to add extra parameters

Comment (sergi-hmrc) [reply app/views/includes/forms/card.scala.html:42]: isn't the intention to always have a list? then why do we keep a list and a body content? 
Shouldn't it just be `bodyContentList: Seq[String] = Seq.empty`

Comment (sergi-hmrc) [reply app/views/includes/forms/card.scala.html:42]: for now, and as @brendanstraughan said, just use nonEmpty. no need to add another parameter

Comment (masahmad) [reply app/views/includes/forms/card.scala.html]: cool updated

**Review by sergi-hmrc — APPROVED** [review]
lgtm

**Comment by vivekhmrc** [comment]
The content suggested on this story has been superseded by another iteration of the prototype so this can be closed.

**Pull request closed without merging**